website/content/en/status/report-2023-04-2023-06/gcc.adoc

```diff
@@ -23,7 +23,7 @@ A few significant changes have been made on the *-devel ports, which will be for
 
 * enable support for .init_array and .fini_array, which FreeBSD supports since commit gitref:83aa9cc00c2d83d05a0efe7a1496d8aab4a153bb[repository=src];
 
-* switch back to STANDARD_BOOTSTRAP on i386, amd64 and aarch64 as default option instead of LTO_BOOTSTRAP: LTO bootstrap produces too many failures on the package builders for those architectures.
+* switch back to STANDARD_BOOTSTRAP on i386, amd64 and aarch64 as the default instead of LTO_BOOTSTRAP: LTO bootstrap produces too many failures on the package builders for those architectures.
```

Comment:
I would like to keep the information that it is an option: how about "as option enabled by default"?